android/src/main/java/com/bitmovin/player/reactnative/services/MediaSessionPlaybackService.kt

```diff
@@ -0,0 +1,55 @@
+package com.bitmovin.player.reactnative.services
+
+import android.content.Intent
+import android.os.Binder
+import android.os.IBinder
+import com.bitmovin.player.api.Player
+import com.bitmovin.player.api.media.session.MediaSession
+import com.bitmovin.player.api.media.session.MediaSessionService
+
+class MediaSessionPlaybackService : MediaSessionService() {
+    inner class ServiceBinder : Binder() {
+        var player: Player?
+            get() = this@MediaSessionPlaybackService.player
+            set(value) {
+                disconnectSession()
+                this@MediaSessionPlaybackService.player = value
+                value?.let {
+                    createSession(it)
+                    connectSession()
+                }
+            }
+    }
+
+    private var player: Player? = null
+    private val binder = ServiceBinder()
+    private var mediaSession: MediaSession? = null
+
+    override fun onGetSession(): MediaSession? = null
```

Comment:
Shouldn't this return the mediaSession instance?

Reply:
As this function is not used anywhere, we decided to 'restrict' its usage by setting it to `null`

Reply:
It's fine returning `null` here as we call `addSession` manually in `connectSession`

Reply:
Ok, thanks!